site/docs/transform/stack.md

```diff
@@ -94,8 +96,47 @@ If you want to define custom sort order, you can derive a new field using the [`
 
 Note: we plan to have [a better syntax for customized sort order](https://github.com/vega/vega-lite/issues/2915) in the future.
 
-## Layering Lines on top of Stacked Area Chart
+### Layering Lines on top of Stacked Area Chart
 
 Since `line` marks are not stacked by default, to layer lines on top of stacked area charts, you have to manually set the `stack` offset for the lines.
 
 <div class="vl-example" data-name="normalized/stacked_area_overlay_normalized"></div>
+
+{:#transform}
+## Stack Transform
+
+{: .suppress-error}
+```json
+// A View Specification
+{
+  ...
+  "transform": [
+    {"stack": ..., "groupby": ..., "offset": ..., "sort": ..., "as" ...} // Stack Transform
+     ...
+  ],
+  ...
+}
+```
+
+The `stack` transform in the `transform` array has the following properties:
+
+{% include table.html props="stack,groupby,offset,sort,as" source="StackTransform" %}
+
+### Diverging Bar Chart
+
+Here we initially `stack` by `"question"` and then use the `window` transform to offset each stack.
+<div class="vl-example" data-name="bar_diverging_stack_transform"></div>
```

Comment:
Is `bar_diverging_stack_transform` the same example as in the gallery?

Reply:
I'll make a separate PR for replacing the gallery example.

Reply:
I think it should be done within the scope of this PR.